packages/contracts/src/test/osx-versions/Migration.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
```

Comment:
We should write a comment here explaining why this file is needed

Reply:
amended

Reply:
+1